ogcore/fiscal.py

```diff
@@ -298,3 +306,51 @@ def get_r_gov(r, p):
     r_gov = np.maximum(p.r_gov_scale * r - p.r_gov_shift, 0.00)
 
     return r_gov
+
+
+def get_I_g(Y, alpha_I):
+    r'''
+    Find investment in public capital
+
+    .. math::
+        I_{g,t} = \alpha_{I,t}Y_{t}
+
+    Args:
+        Y (array_like): aggregate output
+        alpha_I (array_like): percentage of output invested in public capital
+
+    Returns
+        I_g (array_like): investment in public capital
+    '''
+    I_g = alpha_I * Y
+
+    return I_g
+
+
+def get_K_g(K_g0, I_g, p, method):
+    r'''
+    Law of motion for the government capital stock
+
+    .. math::
+        K_{g,t+1} = \frac{(1 - \delta_g)K_{g,t} + I_{g,t}}
+            {(1 + g_{n,t+1})e^{g_y}}
+
+    Args:
+        Y (array_like): aggregate output
+        alpha_I (array_like): percentage of output invested in public capital
+
+    Returns
+        K_g (array_like): stock of public capital
+
+    '''
+    if method == 'TPI':
+        K_g = np.zeros(p.T)
+        K_g[0] = K_g0
+        for t in range(p.T - 1):  # TODO: numba jit this
+            growth = (1 + p.g_n[t + 1]) * np.exp(p.g_y)
+            K_g[t + 1] = ((1 - p.delta_g) * K_g[t] + I_g[t]) / growth
+    else:  # SS
+        growth = (1 + p.g_n_ss) * np.exp(p.g_y)
+        K_g = I_g / (1 - ((1 - p.delta_g) / growth))
```

Comment:
@jdebacker. This expression for steady-state `K_g` is not correct. For it to be correct, the denominator must be multiplied by `growth`. I have made this change in [my PR](https://github.com/jdebacker/OG-Core/pull/24) to your branch. The new line is the following:
```python
K_g = I_g / (growth - (1 - p.delta_g))
```